releasenotes/notes/oxidize-synth_cnot_phase_aam-9c0566d2c2842dbd.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade_synthesis:
+  - |
+    Ported :func:`~.synth_cnot_phase_aam` to rust. The algorithm computes minimal-CNOT
+    circuit for a given phase-polynomial. The newly ported rust code has a speedup of 
```

If I understand correctly, this is a heuristic algorithm and is not guaranteed to produce circuits with the *minimum* number of CX-gates.
```suggestion
    Ported :func:`~.synth_cnot_phase_aam` to Rust, which is a heuristic algorithm 
    for synthesizing small parity networks. The newly ported Rust code has a speedup of 
```